etc/csp_whitelist.xml

```diff
@@ -23,7 +23,9 @@
                 <value id="extend-prod-images" type="host">shipping-offers-static-images-bucket-prod.s3.amazonaws.com</value>
                 <value id="extend-dev-images" type="host">shipping-offers-static-images-bucket-dev.s3.amazonaws.com</value>
                 <value id="extend-demo-images" type="host">shipping-offers-static-images-bucket-demo.s3.amazonaws.com</value>
+                <value id="extend-static-assets" type="host">helloextend-static-assets.s3.amazonaws.com</value>
+                <value id="fallback-images" type="host">s3.amazonaws.com</value>
```

Comment:
IMO we should not be whitelisting entire s3. Basically it will allow any s3 bucket out in the universe/

Reply:
I suppose I agree, are there weird one-off Extend assets that don't fall under one of these hosts?

Reply:
yes there is: https://s3.amazonaws.com/ccspersistenceprod-contentstaticassets04b201d4-1apqb8dyegznm/offers/learnMoreModal-default-1648832726125-learnMoreModal.backgroundImageUrl_Generic_WomanwithBox-min.jpg

I'm just going to include it manually. I am pretty sure it is the only one.